flight/template/View.php

```diff
@@ -130,6 +130,7 @@ public function render($file, $data = null)
         $this->template = $this->getTemplate($file);
 
         if (!file_exists($this->template)) {
+            $this->template = self::normalizePath($this->template);
```

Comment:
So this will fix the path....but then it throws an exception so nothing is really fixed.

Reply:
Do you mean about empty strings?

Reply:
Do you mean about empty strings?